### ISSUE TITLE
day.tasmin suspicious values

### ISSUE THREAD

**Issue by martinjuckes** [cp4cds/cmip6_range_check#18]
SNU_SAM0-UNICON_historical :: Sungsu Park (sungsup@snu.ac.kr), Jihoon Shin (sjh11556@snu.ac.kr)
Min: 139.2387237548828 -- 140.0
CMIP6.CMIP.SNU.SAM0-UNICON.historical.r1i1p1f1.day.tasmin.gn.v20190323 -- [hdl:21.14100/838c237a-beb5-3a95-9552-71b03a51ec45](http://hdl.handle.net/21.14100/838c237a-beb5-3a95-9552-71b03a51ec45)

**Comment by martinjuckes** [comment]
email: June 15th: Dear Martin

Thank you very much for letting us know this problem.

We have re-checked the entire CMIP6 data we produced and found that such a low value of daily minimum temperature occurs only at one single day.
This is because the convection scheme in our model can rarely (one time in the entire hundreds of years) produce a very strong temperature tendency in the lowest model layer. 
Although the value is suspiciously low, it does not affect the overall simulations at all (because it occurs very very rarely).
In addition, we double-checked that all the data was processed correctly.
In summary, it is OK for the community to proceed with our simulation data.

Please let me know how to inform the issue to the community.

Best wishes

Sungsu
